anaconda.spec.in

```diff
@@ -205,6 +205,17 @@ Requires: tmux
 Requires: gdb
 # support for installation from image and live & live image installations
 Requires: rsync
+# basic filesystem tools
+%if ! 0%{?rhel}
+Requires: btrfs-progs
+Requires: ntfs-3g
+Requires: ntfsprogs
+Requires: jfsutils
```

Comment:
I think JFS would be a good candidate for removing from the list of supported filesystems, we probably don't have much users of this ancient AIX filesystem on Fedora, but I don't want to be the one to decide it.